rkt/image_extract.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/coreos/rkt/pkg/fileutil"
+	"github.com/coreos/rkt/store"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/spf13/cobra"
+)
+
+var (
+	cmdImageExtract = &cobra.Command{
+		Use:   "extract IMAGE OUTPUT_ACI_FILE",
+		Short: "extract a stored image to an ACI file",
+		Long:  `IMAGE should be a string referencing an image: either a hash or an image name.`,
+		Run:   runWrapper(runImageExtract),
+	}
+	flagRootfsOnly bool
+	flagOverwrite  bool
+)
+
+func init() {
+	cmdImage.AddCommand(cmdImageExtract)
+	cmdImageExtract.Flags().BoolVar(&flagRootfsOnly, "rootfs-only", false, "extract rootfs only")
+	cmdImageExtract.Flags().BoolVar(&flagOverwrite, "overwrite", false, "overwrite output directory")
+}
+
+func runImageExtract(cmd *cobra.Command, args []string) (exit int) {
+	if len(args) != 2 {
+		cmd.Usage()
+		return 1
+	}
+
```

Comment:
nit, I would rather assign args[1] to a meaningful variable name instead of referencing it directly in a bunch of places